01_fibonacci/main.go

```diff
@@ -0,0 +1,24 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+)
+
+var out io.Writer = os.Stdout
+
+func main() {
+
+	for num := 1; num < 10; num++ {
+		fmt.Fprintln(out, printNumber(num))
+	}
+
+}
+func printNumber(x int) int {
```

Comment:
Your function doesn't actually print anything a new containing `fibonacci` is probably more descriptive.

Reply:
i *like* how this function doesn't print anything -- but i agree it should be named in a way so that the programmer reading the code isn't surprised by the behaviour of the function. A function named `printNumber` that doesnt print anything would be very surprising!

Reply:
Changed the function name